.github/workflows/gclientgen.yaml

```diff
@@ -0,0 +1,75 @@
+name: Client Generator
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+    strategy:
+      matrix:
+        go-version: [ '1.23' ]
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Setup Go ${{ matrix.go-version }}
+      uses: actions/setup-go@v5
+      with:
+        go-version: ${{ matrix.go-version }}
+        cache-dependency-path: |
+          go.work.sum
+          gclientgen/go.sum
+    - name: Display Go version
+      run: go version
+    - run: go -C gclientgen get ./...
+    - run: go -C gclientgen test ./...
```

Comment:
Since we are in a Go workspace we can remove -C flag as the Go command is aware of all declared modules in the Workspace and will run tests when using the `./...` pattern.

Reply:
That does not work for me, maybe because I did something wrong elsewhere?

```
go test ./...
pattern ./...: directory prefix . does not contain modules listed in go.work or their selected dependencies
```

Reply:
I don't think there is a need for a `go.work` file for this module

Reply:
That is fine too, in that case I think you could add a default working directory for the job. Something like

```
defaults:
      run:
        working-directory: ./generator
```